Apps/Sandcastle/gallery/3D Tiles Clipping Planes.html

```diff
@@ -255,7 +255,6 @@
 
 function reset() {
     viewer.entities.removeAll();
-    viewer.scene.primitives.removeAll();
```

Comment:
I think you need to replace this with something like `viewer.scene.primitives.remove(tileset)`

Reply:
Can you make that change to `Terrain Clipping Planes.html` too?